src/types.ts

```diff
@@ -20,19 +20,25 @@ export interface AragonConfig {
 }
 
 export interface AragonConfigHooks {
-  preDao?: (bre: BuidlerRuntimeEnvironment) => Promise<void> | void
+  preDao?: (params: {}, bre: BuidlerRuntimeEnvironment) => Promise<void> | void
```

Comment:
This is the hooks types being forced to have the two parameters mentioned above.